backend/api/Configurations/CustomServiceConfigurations.cs

```diff
@@ -124,5 +125,33 @@ IConfiguration configuration
 
             return services;
         }
+
+        public static IServiceCollection ConfigureMissionLoader(
+            this IServiceCollection services,
+            IConfiguration configuration
+        )
+        {
+            string? missionLoaderFileName = configuration["MissionLoader:FileName"];
+            Console.WriteLine(missionLoaderFileName);
```

Comment:
This should either be removed or logged with a proper logger

Reply:
Removed